tests/test_convert.py

```diff
@@ -10,13 +10,31 @@
 
 
 def test_to_elegant():
+    expected = (
+        "! TITLE: FODO Lattice\n"
+        "M1: MARK, \n"
+        "D1: DRIF, L=0.55\n"
+        "Q1: KQUAD, L=0.2, K1=1.2\n"
+        "Q2: KQUAD, L=0.4, K1=-1.2\n"
+        "B1: CSBEND, L=1.5, ANGLE=0.39269908169872414, E1=0.19634954084936207, "
+        "E2=0.19634954084936207\n"
+        "CELL: LINE=(M1, Q1, D1, B1, D1, Q2, D1, B1, D1, Q1)\n"
+        "RING: LINE=(CELL, CELL, CELL, CELL, CELL, CELL, CELL, CELL)\n"
+        "USE, RING\n"
+    )
+
```

Comment:
In #12, I have added an examples.json file, I would rather use this file than the fodo.json to test the conversion.

Reply:
Nevermind, it is more reasonable to use a "real" lattice file to test if the conversion is working